Misc/NEWS.d/next/Documentation/2018-07-28-12-41-01.bpo-22062.TaN2hn.rst

```diff
@@ -0,0 +1 @@
+documentation and docstrings updated, Original patch by Mike Short
```

Comment:
Nitpick: The blurbs usually use complete sentences – start with a capital letter and put period (not comma) before the author notice. (You can see the [changelog](https://docs.python.org/3/whatsnew/changelog.html#changelog) for examples of how these are formatted.)